CONFIGURATION.md

```diff
@@ -0,0 +1,74 @@
+ORACLE_VALIDATOR_ADDRESS_PRIVATE_KEY | The private key of the bridge validator used to sign confirmations before sending transactions to the bridge contracts. The validator account is calculated automatically from the private key. Every bridge instance (set of watchers and senders) must have its own unique private key. The specified private key is used to sign transactions on both sides of the bridge. | hexidecimal without "0x"
+ORACLE_VALIDATOR_ADDRESS | The public address of the bridge validator | hexidecimal with "0x"
+ERC20_TOKEN_ADDRESS | address of token for ERC-TO-ERC type of bridge | hexidecimal with "0x"
+
+
+## UI configuration
+
+Please note that the common configuration parameters need to be prefixed with `REACT_APP_`, e.g. `REACT_APP_COMMON_HOME_RPC_URL`
+
+name | description | value
+--- | --- | ---
+REACT_APP_TITLE | The title for the bridge UI page. `%c` will be replaced by the name of the network. | string
+REACT_APP_DESCRIPTION | The meta description for the deployed bridge page. | string
+REACT_APP_NATIVE_TOKEN_DISPLAY_NAME | name of the home native coin | string
+REACT_APP_HOME_NETWORK_DISPLAY_NAME | name to be displayed for home network | string
+REACT_APP_FOREIGN_NETWORK_DISPLAY_NAME | name to be displayed for foreign network | string
+REACT_APP_HOME_WITHOUT_EVENTS | `true` if home network doesn't support events | true/false
+REACT_APP_FOREIGN_WITHOUT_EVENTS | `true` if foreign network doesn't support events | true/false
+REACT_APP_HOME_EXPLORER_TX_TEMPLATE | template link to transaction on home explorer. `%s` will be replaced by transaction hash | URL template
+REACT_APP_FOREIGN_EXPLORER_TX_TEMPLATE | template link to transaction on foreign explorer. `%s` will be replaced by transaction hash | URL template
+REACT_APP_HOME_EXPLORER_ADDRESS_TEMPLATE | template link to address on home explorer. `%s` will be replaced by address | URL template
+REACT_APP_FOREIGN_EXPLORER_ADDRESS_TEMPLATE | template link to address on foreign explorer. `%s` will be replaced by address | URL template
+REACT_APP_HOME_GAS_PRICE_UPDATE_INTERVAL | An interval in milliseconds used to get the updated gas price value either from the oracle or from the Home Bridge contract. | integer
+REACT_APP_FOREIGN_GAS_PRICE_UPDATE_INTERVAL | An interval in milliseconds used to get the updated gas price value either from the oracle or from the Foreign Bridge contract. | integer
+PORT | The port for the UI app. | integer
+REACT_APP_STYLES | The set of styles to render the bridge UI page. Currently only `classic` is implemented | classic
+
+
+## Monitor configuration
+
+name | description | value
+--- | --- | ---
+MONITOR_HOME_START_BLOCK | The app will monitor transactions starting from this block | integer
+MONITOR_FOREIGN_START_BLOCK | The app will monitor transactions starting from this block | integer
+MONITOR_VALIDATOR_HOME_TX_LIMIT |  | integer
+MONITOR_VALIDATOR_FOREIGN_TX_LIMIT |  | integer
+MONITOR_TX_NUMBER_THRESHOLD |  | integer
```

Comment:
These 3 variables need some description, it was not described in any documentation before, and I'm not exactly sure what is the purpose for these.

Reply:
`MONITOR_VALIDATOR_HOME_TX_LIMIT` - average gas usage of a transaction sent by a validator, it is used to estimate the number of transaction that can be paid by the validator. 
`MONITOR_VALIDATOR_FOREIGN_TX_LIMIT` - average gas usage of a transaction sent by a validator, it is used to estimate the number of transaction that can be paid by the validator. 
`MONITOR_TX_NUMBER_THRESHOLD` - if estimated number of transaction is equal to or below this value, the monitor will report that the validator has less funds than it is required.